frontend/app/src/components/history/events/query-status/HistoryEventsQueryStatus.vue

```diff
@@ -34,7 +34,7 @@ const {
     </template>
 
     <template #item="{ item }">
-      <LocationIcon icon no-padding :item="item.location" size="20px" />
+      <LocationIcon icon :item="item.location" size="20px" />
```

Comment:
I will keep the minimum for this since the change here just removed the no-padding property but let's create a note in the kanban board of frontend for this.